test/suites/test_dml.py

```diff
@@ -17,6 +18,12 @@ def setUpClass(self):
         self.srv.script = 'test/suites/box.lua'
         self.srv.start()
         self.con = tarantool.Connection(self.srv.host, self.srv.args['primary'])
+        self.con_encoding_utf8 = tarantool.Connection(self.srv.host,
+                                                      self.srv.args['primary'],
+                                                      encoding='utf-8')
+        self.con_encoding_none = tarantool.Connection(self.srv.host,
+                                                      self.srv.args['primary'],
+                                                      encoding=None)
```

Comment:
I wouldn't call it a parametrization since different connections are used in different tests. I'll think about code duplication reduction.

Also, these checks are not about checks -- it's about authentication. I don't know why this approach was chosen to setup connections, but decided not to change existing way of things.

Reply:
Removed code duplication

Reply:
Tests were moved to a separate file